docx/header.py

```diff
@@ -34,6 +35,13 @@ def is_linked_to_previous(self):
             return True
         return False
 
+    @lazyproperty
+    def body(self):
+        """
+        BlockItemContainer instance with contents of Header
+        """
+        return BlockItemContainer(None, self)
```

Comment:
Yeah! The analysis document is handy. I can add the header specimen XML and the new protocol parts.

I like the structure of that `self.part.related_header_body(headerReference.rId)`. Seems like it should work and be testable.

I'm helping give a tutorial at PyCon on Sunday so I'm going to be busy with that the next few days. And then I'll be at PyCon proper. So I probably won't make any forward progress until next Thursday at the earliest.

Are you going to PyCon?